test/test_cartesian_graph.py

```diff
@@ -29,7 +29,7 @@ def test_direct_input_output():
     x = [2.14159]
     y = graph(x)
 
-    assert abs(x[0] - y[0]) < 1e-15
+    assert pytest.approx(x[0] == y[0])
```

Comment:
interesting, i thought one has to write `x[0] == pytest.approx(y[0])`, but of course this is much nicer! :+1: